mlx4_core: big LSO support preparation for packets larger than 64B

Signed-off-by: Ali Ayoub <ali@mellanox.com>

Index: ofed_kernel/include/linux/mlx4/device.h
===================================================================
--- ofed_kernel.orig/include/linux/mlx4/device.h	2010-03-09 15:16:13.593871000 +0200
+++ ofed_kernel/include/linux/mlx4/device.h	2010-03-09 16:41:47.974898000 +0200
@@ -124,6 +124,7 @@ enum {
 	MLX4_OPCODE_SEND		= 0x0a,
 	MLX4_OPCODE_SEND_IMM		= 0x0b,
 	MLX4_OPCODE_LSO			= 0x0e,
+	MLX4_OPCODE_BIG_LSO		= 0x2e,
 	MLX4_OPCODE_RDMA_READ		= 0x10,
 	MLX4_OPCODE_ATOMIC_CS		= 0x11,
 	MLX4_OPCODE_ATOMIC_FA		= 0x12,
